modules/core/arrow-test/src/main/kotlin/arrow/test/generators/LiftGen.kt

```diff
@@ -0,0 +1,54 @@
+package arrow.test.generators
+
+import arrow.Kind
+import arrow.core.ForId
+import arrow.core.ForListK
+import arrow.core.ForNonEmptyList
+import arrow.core.ForOption
+import arrow.core.ForSequenceK
+import arrow.core.Id
+import arrow.core.ListK
+import arrow.core.NonEmptyList
+import arrow.core.Option
+import arrow.core.SequenceK
+import io.kotlintest.properties.Gen
+
+interface LiftGen<F> {
+  /**
+   * lifts a Gen<A> to the context F. the resulting Gen can be used to create types Kind<F, A>
+   */
+  fun <A> liftGen(gen: Gen<A>): Gen<Kind<F, A>>
+}
```

Comment:
I opened a ticket because I noticed some broken tests with the current way we would encode this pattern: https://github.com/arrow-kt/arrow/issues/1819
This is precisely how I think that should be solved :+1: 
To keep this more in line with the current naming, it might make sense to name it `GenK` tho, but other than that this is great!

Reply:
cool!